### PR TITLE
can-wait should work when xhr event is not provided.

### PR DESCRIPTION
**Pull request by m-mujica**
cc: @matthewp

### CODE REVIEW

**Comment by matthewp** [comment]
Thanks, this is needed for the Node XHR polyfill, it doesn't provide the event object (until https://github.com/flububb/node-XMLHttpRequest/pull/1 is merged/released).

File: test/test.js [hunk 1]
```diff
@@ -124,6 +124,42 @@ describe("setTimeout and XHR", function(){
 
 			}).then(done);
 		});
+
+		describe("if event is not provided", function() {
```
Comment (matthewp) [inline test/test.js]: change this to `if event is not provided to onreadystatechange` to clarify.

**Comment by m-mujica** [comment]
@matthewp I changed the test description.

**Comment by matthewp** [comment]
Released as can-wait 0.2.11

**Pull request merged**